### PR TITLE
Update Elasticsearch docs, KB article

### LINKED ISSUE #1879: Update AWS Elasticsearch docs to be more helpful
Right now, our ES docs are really focused on upgrading from the old ES service, which is really confusing to users who didn't use that service. We also defer a lot to the AWS docs, which means getting things done requires hopping back and forth between pages. 

We should add documentation in tutorial and explanation form to:
- create a service instance
- interact with a service instance from a local command line, including where to get credentials from and spelling out what values a user needs to define themselves (n.b. because of the aws-header stuff, this will probably have to recommend using something like the python or ruby REPL)

We should also document directly that Kibana is not included in this offering, and that it's probably incompatible with this service.

### PR DESCRIPTION
**Pull request by mheadd**
## Changes proposed in this pull request:
- Clarify that ES service is not suitable for running your own logging stack
- Clarify service creation example
- Misc cleanup
- Fixes #1879 

:sunglasses: [PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/mheadd-patch-24/docs/services/aws-elasticsearch/)

## Security Considerations
None

### CODE REVIEW

File: _docs/services/aws-elasticsearch.md [hunk 1]
```diff
@@ -69,10 +68,9 @@ $ cf create-service aws-elasticsearch es-medium my-es-service-3 -c '{"advanced_o
 
 ```
 
-
 ### Shard/replica configuration for high availability
```
Comment (mheadd) [inline _docs/services/aws-elasticsearch.md:73]: @ChrisMcGowan Does this section still make sense?

File: _docs/services/aws-elasticsearch.md [hunk 2]
```diff
@@ -137,7 +142,3 @@ Note: If you have access to larger service plans, they will mirror same settings
 ## Rotating credentials
 
 You can rotate credentials by creating a new instance and [deleting the existing instance](https://cli.cloudfoundry.org/en-US/cf/delete-service.html). If this is not an option, email [cloud.gov support](mailto:support@cloud.gov) to request rotating the credentials manually.
-
```
Comment (mheadd) [inline _docs/services/aws-elasticsearch.md:140]: Do we really need this on the docs page?

Comment (pburkholder) [reply _docs/services/aws-elasticsearch.md:140]: The link to GitHub I think is a good thing for those interested in a deep dive. 

Is credential rotation really that hard?

File: _docs/services/aws-elasticsearch.md [hunk 3]
```diff
@@ -32,11 +24,21 @@ Service Name | Plan Name | Description | Number of nodes |
 
 $200/month per node for **medium**, $400/month per node for **large**. Six nodes included for customers using the FISMA Moderate plan. More information on the [pricing page]({{ site.baseurl }}{% link _pages/pricing.md %}).
 
+## When to use
+
+This service is geared toward applications that need to provide search capability, or interact with indexed data. Although Elasticsearch is often used as part of the ELK logging stack (Elasticsearch + Logstash + Kibana), this service **does not** include Kibana, and isn't suitable as a component of a logging solution. To find out more about logging on the cloud.gov platform, please see [the section on logs]({{ site.baseurl }}/docs/deployment/logs/#web-based-logs-with-historic-log-data).
```
Comment (bengerman13) [inline _docs/services/aws-elasticsearch.md:29]: 👨‍🍳 👌

File: _docs/services/aws-elasticsearch.md (hunk 2 above)
Comment (bengerman13) [reply _docs/services/aws-elasticsearch.md:140]: the github link is a thing we include on most or all of our other brokers, and I think it's a good reminder

Comment (mheadd) [reply _docs/services/aws-elasticsearch.md:140]: Added this back

**Review by soutenniza — APPROVED** [review]
+1

**Pull request merged**